packages/web/DEPRECATIONS.md

```diff
@@ -27,6 +27,10 @@ The direction values `row` and `column` were removed, please use `horizontal` an
 
 The `Header` component was removed, please use `UNSTABLE_Header` component instead.
 
+### Skeleton
+
+The check for the existence of the `skeleton-gradient` token will be removed. Ensure that the skeleton-gradient token is properly set up in your project, as the project will not run without it.
```

Comment:
> the project will not run without it.

It depends; it won't run if you import all components and not specific ones.

Reply:
Once fixed, please sync the changes to `Skeleton` README.